### PR TITLE
added two simple charts

### PR DESCRIPTION
**Pull request by CMRHDL**
Added two simple Charts within a new page. Sadly its build onto the old navbar, so hopefully its easy to merge respectively refactor.
Added charts.js, angular-charts.js and lodash.js.
Not sure about licenses ...
https://github.com/nnnick/Chart.js/blob/master/LICENSE.md
https://github.com/lodash/lodash/blob/master/LICENSE
https://github.com/jtblin/angular-chart.js/blob/master/LICENSE

### CODE REVIEW

**Comment by sepe81** [comment]
Looks very promising. I will try it out soon... 😀

File: bower.json [hunk 1]
```diff
@@ -20,6 +20,8 @@
     "angularjs": "~1.4.0",
     "angular-route": "~1.4.0",
     "angular-ui-bootstrap": "~0.13.0",
-    "bootstrap": "~3.3.2"
+    "bootstrap": "~3.3.2",
+    "angular-chart.js": "~0.7.2",
+    "lodash": "^3.9.3",
```
Comment (sepe81) [inline bower.json]: @CMRHDL please remove the superfluous comma, otherwise "bower install" fails

**Comment by sepe81** [comment]
@CMRHDL the option "show km per day" already looks nice, but the other two options (for year/month) don't work. The overview shows a circle with the wrong number of runs and has an a strange animation while opening the view. Any ideas on that?

**Comment by CMRHDL** [comment]
charts

Overview consists of all logged activities and shows distribution. In case you only logged running so far, its not very useful I guess. Not sure what happens with the animation in that case.
I also noticed while trying to create testdata that you cant manually add "old" activities, they wont get recognised for some reason. Maybe thats why your overall data count doesnt match.

Concerning "show km per day", since I wasnt able to create testdata I cant really debug. Maybe the API cant handle too many labels/values/overall width. Working fine locally with two testvalues.

**Comment by sepe81** [comment]
@CMRHDL I could fix the merge conflict in bower.json through first reverting some conflichanges in master branch.

The "problem" with the uncomplete diagram is caused by the missing size parameter in ActivityService.list(). The list function needs a size param otherwise the default pagination just loads the 10 newest activities. I'm going to fix this in a new PR.

**Pull request merged**